draft-ietf-rats-corim.md

```diff
@@ -1595,17 +1602,18 @@ Profile (label 6):
 ~~~ cddl
 ECT = {
   ? e: environment-map
-  ? c: claims-map / [ + local-claim ]
+  ? c: [ + local-claim ]
   ? a: [ + $crypto-key-type-choice ]
   ? ns: text
   ? cm: cm-type
   ? p: $profile-type-choice
 }
 local-claim = {
-  le: local-environment
+  ? eid: element-id
   c: claims-map
 }
-local-environment =  bstr / tstr
+element-id =  bstr / tstr / uint / tagged-oid-type / tagged-uuid-type
```

Comment:
Note: element-id could go back to being bstr / tstr if the mapping from external to internal describes a mapping from uint, tagged-oid-type and tagged-uuid-type to bstr.